components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -98,19 +162,23 @@ const UserSettingsPage = () => {
                 />
               ))}
             </div>
-            <button className="px-4 w-max py-2  rounded-lg bg-light-slate-4 border border-light-slate-8">
+            <Button type="default" disabled={selectedInterest.length === 0} onClick={handleUpdateInterest} className="!px-4 !text-light-slate-11 !py-2  !bg-light-slate-4">
               Update Interests
-            </button>
+            </Button>
           </div>
           <div className="flex flex-col gap-6">
             <div className="flex flex-col gap-3 ">
               <label className="text-light-slate-11 text-2xl  font-normal">Email Preferences</label>
               <Checkbox value={"true"} title="profile email" label="Display Email On Profile" />
```

Comment:
```suggestion
              <Checkbox value={"false"} title="profile email" label="Display email on profile" />
```

- This should be false by default
- Change to sentence case